common/utils/src/main/resources/error/error-classes.json

```
@@ -2250,6 +2265,11 @@
     },
     "sqlState" : "42703"
   },
+  "UNRESOLVED_EXPRESSION_IN_MERGE_COMMAND_COLUMNS" : {
+    "message" : [
+      "Cannot resolve <expr> in MERGE command given columns [<cols>]."
```

```suggestion
      "Cannot resolve <expr> in the MERGE command. Did you mean one of the following? [<proposal>]."
```
BTW, cannot your re-use the existing error class? `UNRESOLVED_COLUMN.WITH_SUGGESTION`?